docker/Rprofile.site

```diff
@@ -1,2 +1,4 @@
-options(repos = c(CRAN = "https://cloud.r-project.org"),
-        download.file.method = "libcurl")
+options(repos = c(CRAN = 'https://packagemanager.rstudio.com/all/__linux__/focal/latest'), download.file.method = 'libcurl')
```

Comment:
This is actually the same as what is present in the current version of the r-ver image but unless you track latest they fall back on a version pinned to the "last cran update before the release of the following version" which causes all sorts of problems